<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QFrame" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(20, 20, 20);</string>
  </property>
  <widget class="QWidget" name="chat_history" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>900</width>
     <height>1080</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <widget class="QPushButton" name="newChat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>461</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 20, 20);
color: rgb(150, 150, 150);
border-radius: 12px;</string>
    </property>
    <property name="text">
     <string>Start a new chat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newChat2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 20, 20);
border-radius: 12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/comment.png</normaloff>../icons/comment.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QScrollArea" name="historyScrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>481</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
border-style: none;</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>1000</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QScrollArea" name="chatScrollArea">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>1400</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: hidden;</string>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1400</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="messageContainer" native="true">
   <property name="geometry">
    <rect>
     <x>509</x>
     <y>910</y>
     <width>1401</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-color: rgb(50, 50, 50);
border-width: 1px;
border-radius: 12px</string>
   </property>
   <widget class="QPushButton" name="attach">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/attach-file.png</normaloff>../icons/attach-file.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="send">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/paper.png</normaloff>../icons/paper.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="voicemail">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/voice (1).png</normaloff>../icons/voice (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="message">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>1200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: none;
color: rgb(250, 250, 250);
</string>
    </property>
    <property name="placeholderText">
     <string>Type your message here</string>
    </property>
   </widget>
   <widget class="QLabel" name="timerLabel">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>50</y>
      <width>50</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(50, 50, 50);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;00:00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="caution">
   <property name="geometry">
    <rect>
     <x>508</x>
     <y>1000</y>
     <width>1401</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 100, 100);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;The information provided in this chat are not supposed to be strictly accurate.&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Janine helps you  the best she can but is not always right.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_">
   <property name="geometry">
    <rect>
     <x>1860</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: none;
border-radius: 20px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/remove-button.png</normaloff>../icons/remove-button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="pathContainer" native="true">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>850</y>
     <width>1401</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-color: rgb(50, 50, 50);
border-width: 0px;
border-radius: 12px</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../rsrc/images/backgrounds.qrc"/>
 </resources>
 <connections/>
</ui>
